bindings/python/iree/runtime/function_test.py

```diff
@@ -197,6 +198,26 @@ def invoke(arg_list, ret_list):
     with self.assertRaisesRegexp(ValueError, "specified kwarg 'c' is unknown"):
       result = invoker(-1, a=1, b=2, c=3)
 
+  # TODO: Fill out all return types.
+  def testReturnTypeNdArrayBool(self):
```

Comment:
You didn't create the testing deficit, but since you're here, would you mind stamping a variant of this out for all of the supported dtypes (probably except bf16 which no one knows what to do with)? Fine to say now... just trolling for test coverage :)

Reply:
Thanks for the review!

I actually asked on internal chat for clarification on this comment:  I was wondering if the comment meant test type coercion from other ABI types specifically to i1 (say, doing an `i32` to `bool` test), or type coercion from any DTYPE to any type in the ABI (`i32` to `i8`)?

Also, there are some types listed in the `VM_TO_PYTHON_CONVERTERS` that are missing from the `ABI_TYPE_TO_DTYPE`, such at `f16`. Not sure how to handle this (or ignore it)?